Source/Core/DolphinQt/Debugger/CodeDiffDialog.h

```diff
@@ -0,0 +1,67 @@
+// Copyright 2021 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <QDialog>
+
+#include "Common/CommonTypes.h"
+#include "Core/PowerPC/Profiler.h"
+
+class CodeWidget;
+class QCheckBox;
+class QLabel;
+class QListWidget;
+
+struct Diff
+{
+  u32 addr;
+  std::string symbol;
+  u32 hits;
+};
+
+class CodeDiffDialog : public QDialog
+{
+  Q_OBJECT
+
+  struct _AddrOp
+  {
+    bool operator()(const Diff& iter, const std::string& val) const { return iter.symbol < val; }
+  } AddrOP;
+
+public:
+  explicit CodeDiffDialog(CodeWidget* parent);
+  void reject() override;
+
+private:
+  void CreateWidgets();
+  void ConnectWidgets();
+  void ClearData();
+  void ClearBlockCache();
+  void OnRecord(bool enabled);
+  std::vector<Diff> CalculateSymbolsFromProfile();
+  void OnInclude();
+  void OnExclude();
+  void RemoveMatchingSymbolsFromIncludes(std::vector<Diff>& symbol_list);

```

Comment:
The parameter should be passed as const reference:
```c++
void RemoveMatchingSymbolsFromIncludes(const std::vector<Diff>& symbol_list)
```

Reply:
Violates algorithm.h, not changing

Reply:
Ditto.

Reply:
Fixed (see MissingSymbols func)